images/ceph/build-ceph.sh

```diff
@@ -17,7 +17,7 @@
 CEPH_INSTALL_DIR=${CEPH_INSTALL_DIR:-/build/ceph-install}
 CEPH_TARBALL=${CEPH_INSTALL_DIR}.tar
 
-makeargs=-j$(( $(nproc) - 1 ))
+makeargs=-j$(nproc)
```

Comment:
could we still subtract 1 if there are at least 2 procs? Seems like a good thing to not use all cpus on the machine

Reply:
we might be removing the whole process of building ceph in our images. see #1344. this PR might not be needed.

Reply:
@bassam Still users currently can't build Ceph on machines with just one CPU and need to manually intervene.
I don't see it as a problem to fix it, even when it is removed soon so the issue is resolved.